paymentsheet/src/main/java/com/stripe/android/link/ui/wallet/WalletViewModel.kt

```diff
@@ -247,7 +261,8 @@ internal class WalletViewModel @Inject constructor(
             LinkConfirmationResult.Succeeded -> {
                 dismissWithResult(
                     LinkActivityResult.Completed(
-                        linkAccountUpdate = LinkAccountUpdate.Value(null)
+                        linkAccountUpdate = LinkAccountUpdate.Value(linkAccount),
```

Comment:
This can stay unchanged, no?

Reply:
Actually this would clear the link account value unintentionally:

- `LinkAccountUpdate.None`: The result receiver should not update the current account value
- `LinkAccountUpdate.Value(account)`: The result receiver should update the current account value to `account`: This is what we want here, so that FlowController activities can update the link account value post authentication in the Link flow.  
- `LinkAccountUpdate.Value(null)`: The result receiver should update the current account value to `null`